src/styles/dimensions.styles.ts

```diff
@@ -0,0 +1,6 @@
+import { Dimensions } from "react-native";
+
+export const screenHeight = Dimensions.get("window").height;
+export const screenWidth = Dimensions.get("window").width;
+export const screenRatio = screenWidth / screenHeight;
+export const buildingInformationHeight = Dimensions.get("window");
```

Comment:
This new variable is kinda duplicating what the other ones are doing and the name is a bit confusing. Consider deleting it, removing [this line](https://github.com/RGPosadas/WayFinder/pull/91/files#diff-95febc3241bd6b6f610d17f5f4355b4bR22) and importing/using screenHeight instead of height in `src/components/building-information/building-information.component.tsx`

Reply:
addressed the changes